crates/biome_cli/tests/snapshots/main_cases_editorconfig/should_emit_diagnostics.snap

```diff
@@ -0,0 +1,42 @@
+---
+source: crates/biome_cli/tests/snap_test.rs
+expression: content
+---
+## `.editorconfig`
+
+```editorconfig
+
+[*]
+max_line_length = off
+insert_final_newline = false
+
+```
+
+## `test.js`
+
+```js
+console.log("foo");
+
+```
+
+# Emitted Messages
+
+```block
+configuration ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  × Key 'insert_final_newline' is incompatible with biome: Biome always inserts a final newline.
+  
+
+```
+
+```block
+configuration ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  × Key 'max_line_length' is incompatible with biome: 'off' is incompatible with how Biome works.
```

Comment:
The spec says that `off` means "use the default settings". Can't we do that?

Regarding the message, I think we could be more specific. Saying "incompatible with Biome" seems to be too generic. Also, the message isn't actionable, because it doesn't tell the user what they should do to fix the issue.

Reply:
Yeah we can do that.

Btw, the only mention of "off" seems to be in this document: https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties and it doesn't seem to be present in https://spec.editorconfig.org/

Reply:
Yeah I noticed that too 🥲

Reply:
Perhaps they standardized `off` to `unset`?